.github/workflows/ci.yaml

```
@@ -28,12 +28,21 @@ jobs:
       helm_changed: ${{ steps.helm.outputs.any_changed }}
       prerequisites_changed: ${{ steps.prerequisites.outputs.any_changed }}
       ci_changed: ${{ steps.ci.outputs.any_changed }}
+      run_anyway: ${{ steps.set-run-anyway.outputs.run_anyway }}
```

```suggestion
      run_anyway: ((contains("release-"), github.base_ref) || github.ref == 'refs/heads/main'
```
if I'm not mistake you can just do it like this, without bash

---

oh cool, so it only works for `if`, so yeah the way you implemented and if it works let's go this way.